.github/workflows/release.yml

```diff
@@ -0,0 +1,316 @@
+                  cmake --build SFML-build --config Release --target install
+
+            - name: Build CSFML
+              run: |
+                  cmake -S CSFML -B CSFML-build -A x64 -G "Visual Studio 17 2022" -DBUILD_SHARED_LIBS=TRUE -DSTATIC_STD_LIBS=TRUE -DCSFML_LINK_SFML_STATICALLY=TRUE -DCMAKE_BUILD_TYPE=Release -DSFML_DIR=${{ github.workspace }}/SFML-install/lib/cmake/SFML -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/CSFML-install -DCMAKE_VERBOSE_MAKEFILE=ON
+                  cmake --build CSFML-build --config Release --target install
+
+            - name: Add MinGW to PATH
+              shell: bash
+              run: echo "${{ github.workspace }}\mingw64\bin" >> $GITHUB_PATH
+
+            - name: Generate Import Libs for MinGW
+              shell: pwsh
+              run: |
+                  New-Item -ItemType "directory" -PATH "CSFML-install\lib\gcc"
+                  New-Item -ItemType "directory" -PATH "CSFML-install\lib\msvc"
+                  Move-Item -Path "CSFML-install\lib\*.lib" -Destination "CSFML-install\lib\msvc"
+
+                  Push-Location CSFML-install\lib\gcc
+                  & gendef.exe "..\..\bin\csfml-graphics-2.dll"
+                  & dlltool.exe -d "csfml-graphics-2.def" -D "..\..\bin\csfml-graphics-2.dll" -l "libcsfml-graphics.a"
+                  Remove-Item -Path "csfml-graphics-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-window-2.dll"
+                  & dlltool.exe -d "csfml-window-2.def" -D "..\..\bin\bin\csfml-window-2.dll" -l "libcsfml-window.a"
+                  Remove-Item -Path "csfml-window-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-audio-2.dll"
+                  & dlltool.exe -d "csfml-audio-2.def" -D "..\..\bin\bin\csfml-audio-2.dll" -l "libcsfml-audio.a"
+                  Remove-Item -Path "csfml-audio-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-network-2.dll"
+                  & dlltool.exe -d "csfml-network-2.def" -D "..\..\bin\bin\csfml-network-2.dll" -l "libcsfml-network.a"
+                  Remove-Item -Path "csfml-network-2.def"
+                  
+                  & gendef.exe "..\..\bin\csfml-system-2.dll"
+                  & dlltool.exe -d "csfml-system-2.def" -D "..\..\bin\bin\csfml-system-2.dll" -l "libcsfml-system.a"
+                  Remove-Item -Path "csfml-system-2.def"
+                  Pop-Location
+
+                  New-Item -ItemType "directory" -PATH "install"
+                  Move-Item -Path "CSFML-install\*" -Destination "install\CSFML"
+
+            - name: Upload Artifact
+              uses: actions/upload-artifact@v4
+              with:
+                  name: CSFML-windows-64-bit
+                  path: install/CSFML
+
+    macos-x64:
```

Comment:
Intel Macs haven't been relevant for a while so I think we're safe to drop this hardware config at any time. Not much harm in keeping it but also not a ton of value in keeping it.

Reply:
I disagree 😜

Reply:
> sent from my Intel Mac Mini